packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3810,13 +3810,62 @@ typedef ErrorWidgetBuilder = Widget Function(FlutterErrorDetails details);
 /// where the problem lies. Exceptions are also logged to the console, which you
 /// can read using `flutter logs`. The console will also include additional
 /// information such as the stack trace for the exception.
+///
+/// It is possible to override this widget.
+///
+/// {@tool snippet --template=freeform}
+/// ```dart
+/// import 'package:flutter/material.dart';
+///
+/// void main() {
+///   ErrorWidget.builder = (FlutterErrorDetails details) {
+///     bool inDebug = false;
+///     assert(() { inDebug = true; return true; }());
+///     // In debug mode, use the normal error widget which shows
+///     // the error message:
+///     if (inDebug)
```

Comment:
Just use `if (!kReleaseMode)`?

https://master-api.flutter.dev/flutter/foundation/kReleaseMode-constant.html

Reply:
That would make profile and release mode different, no?

Reply:
`if (!kReleaseMode && !kProfileMode)` :)

Reply:
What if we add a third non-debug mode?

Reply:
(bear in mind this is developer code, so we can't go and change it)